### PR TITLE
Add benchmark suite in benchmark test

### PR DESCRIPTION
**Pull request by Lewuathe**
#### Description

MISC

It added the following benchmark suites. 
- Batch normalization
- Pooling operation
- Reduction operation
- Unary operation

You can only run the specific benchmark suite with `--grep`.

```
$ yarn benchmark --grep='unary(avg)'
```

The default benchmark suites without any `--grep` option was unchanged. In short, `matmul`, `conv2d` and `mobilenet_v1` will run.

---

##### For repository owners only:

Please remember to apply all applicable tags to your pull request.
Tags: FEATURE, BREAKING, BUG, PERF, DEV, DOC, SECURITY

For more info see: https://github.com/tensorflow/tfjs/blob/master/DEVELOPMENT.md

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-core/1284)

### CODE REVIEW

**Comment by Lewuathe** [comment]
Okay. I just want to compare the performance of other ops (batchnormalization etc) because it must be useful when we try to improve these corresponding ops. And also the benchmark implementations for these ops are already available. I thought it is better to make use of them in benchmark tool.
